third_party/2and3/decorator.pyi

```diff
@@ -0,0 +1,90 @@
+# Stubs for decorator (Python 2 and 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+import sys
+from typing import Any, Callable, Dict, List, NamedTuple, Optional, Pattern, Text, Tuple, TypeVar
+
+_C = TypeVar("_C", bound=Callable[..., Any])
+_Func = TypeVar("_Func", bound=Callable[..., Any])
+_Caller = TypeVar("_Caller", bound=Callable[..., Any])
+
+def get_init(cls): ...
+
+# Duplicated from `stdlib/3/inspect.pyi`
+FullArgSpec = NamedTuple('FullArgSpec', [('args', List[str]),
+                                         ('varargs', Optional[str]),
+                                         ('varkw', Optional[str]),
+                                         ('defaults', tuple),
+                                         ('kwonlyargs', List[str]),
+                                         ('kwonlydefaults', Dict[str, Any]),
+                                         ('annotations', Dict[str, Any]),
+                                         ])
+
+if sys.version_info >= (3,):
+    iscoroutinefunction: Callable[[Callable[..., Any]], bool] = ...  # really inspect.iscoroutinefunction
+    getfullargspec: Callable[[object], FullArgSpec] = ...  # really inspect.getfullargspec
```

Comment:
Just use `from inspect import iscoroutinefunction as iscoroutinefunction, getfullargspec as getfullargspec` here to re-export those.

Reply:
iirc I tried this and it caused this error:

```
/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/decorator.py:47: error: Revealed type is 'def (func: builtins.object) -> Tuple[builtins.list[builtins.str], Union[builtins.str, None], Union[builtins.str, None], builtins.tuple[Any], builtins.list[builtins.str], builtins.dict[builtins.str, Any], builtins.dict[builtins.str, Any], fallback=inspect.FullArgSpec]'
/usr/local/Cellar/python@2/2.7.14_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/decorator.py:56: error: All conditional function variants must have identical signatures
```

the types are almost identical but they're being resolved as `inspect.FullArgSpec` and `decorator.FullArgSpec`. That's why I had to redefine the py3 version as `Callable[[object], FullArgSpec]` -- now its type is `decorator.FullArgSpec` instead.

Do you have an idea for a fix? Could a `cast` work?